js/src/ui/Form/InputAddress/inputAddress.js

```diff
@@ -76,7 +76,7 @@ class InputAddress extends Component {
 
     const props = {};
 
-    if (!readOnly && !disabled) {
+    if (!disabled) {
```

Comment:
Just tested, fine as it is. It wasn't properly adding the right underline line on focus, and would break the tabbing.